pkg/dependency/parser/java/pom/pom.go

```diff
@@ -324,7 +324,7 @@ func (props *properties) UnmarshalXML(d *xml.Decoder, _ xml.StartElement) error
 			return xerrors.Errorf("XML decode error: %w", err)
 		}
 
-		(*props)[p.XMLName.Local] = p.Value
+		(*props)[p.XMLName.Local] = strings.TrimSpace(p.Value)
```

Comment:
This change removes spaces only for `properties`.
But `mvn` removes spaces for any field (see #7658 (comment))

Reply:
Right, the latest version trims all fields.